Makefile

```diff
@@ -45,9 +45,11 @@ upgrade_test:
 	make -C ./scripts/feature_tests/upgrade/
 
 feature_tests: setup_env remediation_test cleanup_env pivoting_test
+	./scripts/fetch_manifests.sh
```

Comment:
would it be more consistent if you define a target,
fetch_manifests:
        ./scripts/fetch_manifests.sh

and then use the 'fetch_manifests' target
feature_tests: fetch_manifests setup_env remediation_test cleanup_env pivoting_test

Reply:
Yes that can be done also.

Reply:
Done